.travis.yml

```diff
@@ -42,28 +42,31 @@ matrix:
           env: BUILD=tests,wheels PYTHON_VERSION=3.6.5
           branches: {only: [releases]}
 
+        - os: osx
+          osx_image: xcode7.3
+          # Travis macOS env does not support Python yet,
+          # so we have to set things up manually in install.sh.
+          env: BUILD=tests,wheels PYTHON_VERSION=3.7.0
+          branches: {only: [releases]}
+
         - os: linux
           dist: trusty
-          sudo: false
+          sudo: required
           language: python
           python: "3.5"
-          env: BUILD=tests
+          env: BUILD=tests,wheels,release
 
         - os: linux
           dist: trusty
           sudo: false
           language: python
           python: "3.6"
-          env: BUILD=tests
 
         - os: linux
-          dist: trusty
-          branches: {only: [releases]}
+          dist: xenial
           sudo: required
           language: python
-          python: "3.5"
-          services: [docker]
-          env: BUILD=tests,wheels,release
+          python: "3.7"
```

Comment:
Eh, we needed to keep `env: BUILD=tests`. Never mind, I'll make a PR myself.

Reply:
:+1: reverting my change, it will be better to refactor test after this